testhelper/etcd/etcd.go

```diff
@@ -0,0 +1,67 @@
+package etcd
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ory/dockertest/v3"
+	etcd "go.etcd.io/etcd/client/v3"
+	"google.golang.org/grpc"
+)
+
+type cleaner interface {
+	Cleanup(func())
+	Log(...interface{})
+}
+
+type Resource struct {
+	Client *etcd.Client
+	Hosts  []string
+	Port   int
+}
+
+func Setup(pool *dockertest.Pool, cln cleaner) (*Resource, error) {
+	container, err := pool.Run("bitnami/etcd", "3.4", []string{
```

Comment:
This is not arm64 friendly, correct?

Reply:
I just moved the existing code into this helper because now we're using it in more than one place but I didn't check whether it is arm64 friendly :slightly_smiling_face: